acceptance/auth/bundle_and_profile/output.txt

```diff
@@ -0,0 +1,32 @@
+
+=== Inside the bundle, no flags
+>>> errcode $CLI current-user me
+"$USERNAME"
+
+=== Inside the bundle, target flags
+>>> errcode $CLI current-user me -t dev
+"$USERNAME"
+
+=== Inside the bundle, target and matching profile
+>>> errcode $CLI current-user me -t dev -p DEFAULT
```

Comment:
In addition to Badness in test.toml, can you also add Badness: ..explanation.. to the specific runs where the result is unexpected.